server.js

```diff
@@ -0,0 +1,59 @@
+const express = require('express');
+const product_info = require('./src/data/products.json');
+const cat_info = require('./src/data/categories.json');
+
+// create new express app and save it as "app"
+const app = express();
+
+// server configuration
+const PORT = 8080;
+
+//combining the two json files into a single json object
+const CombineJson = () => {
```

Comment:
Function names should start with lowercase. Rename this to something more meaningful e.g. `combinedProductsData`

Reply:
Done! :)